tests/Services/GetPreliminaryInvoiceTest.php

```diff
@@ -44,6 +44,7 @@ protected function setUp(): void
      */
     public function testGetBill()
     {
+        $this->markTestSkipped();
```

Comment:
ja.... :( Jeg kan ikke lige for nuværende gennemskue, hvordan jeg får mock'et det, som den har tænkt sig at hente fra DB

Reply:
Det er i virkeligheden et signal om, at designet er forkert.
"Design" er måske også et stort ord for den her klasse, hvor alting er blevet skovlet ind.

Reply:
Sådan :) Det skal jo bare gøres ordentligt